ios/CredentialsManagerBridge.swift

```diff
@@ -82,9 +82,42 @@ public class CredentialsManagerBridge: NSObject {
         resolve(credentialsManager.clear())
     }
     
-    @objc public func enableLocalAuthentication(title: String?, cancelTitle: String?, fallbackTitle: String?) {
+    @objc public func enableLocalAuthentication(title: String?, cancelTitle: String?, fallbackTitle: String?, evaluationPolicy: NSNumber?) {
```

Comment:
There's no need for `evaluationPolicy` to be a `NSNumber` instead of an `Int`, I think.